src/usr/lib/rsetup/tui/overlay/overlay.sh

```diff
@@ -137,19 +137,26 @@ __overlay_manage() {
 
     disable_overlays
 
-    local item
+    local items
     for i in "${RSETUP_CHECKLIST_STATE_NEW[@]}"
     do
-        item="$(checklist_getitem "$i")"
-        mv "$U_BOOT_FDT_OVERLAYS_DIR/$item.disabled" "$U_BOOT_FDT_OVERLAYS_DIR/$item"
+        items+=("$(checklist_getitem "$i")")
     done
-
-    if u-boot-update >/dev/null
-    then
-        msgbox "Selected overlays will be enabled at next boot."
-    else
-        msgbox "Unable to update the boot config."
-    fi
+    enable_overlays "${items[@]}"
+    case $? in
```

Comment:
```suggestion
    enable_overlays "${items[@]}"; ret="$?"
    case "$ret" in
```
我们有`set -e`，如果报错会直接终止执行